Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1453,11 +1453,13 @@ section MeasureSpace
 
 variable {s : Set α} [MeasureSpace α] {p : α → Prop}
 
-instance Subtype.measureSpace : MeasureSpace (Subtype p) :=
-  { Subtype.instMeasurableSpace with
-    volume := Measure.comap Subtype.val volume }
+/-- In a measure space, one can restrict the measure to a subtype to get a new measure space. -/
+def Subtype.measureSpace : MeasureSpace (Subtype p) :=
+  { volume := Measure.comap Subtype.val volume }
```

Comment:
Since you're touching this
```suggestion
def Subtype.measureSpace : MeasureSpace (Subtype p) where
  volume := Measure.comap Subtype.val volume
```